include/effects/matrix/spectrumeffects.h

```diff
@@ -292,11 +292,11 @@ class SpectrumAnalyzerEffect : public LEDStripEffect, virtual public VUMeterEffe
                 float agePercent = (float) msPeakAge / (float) MS_PER_SECOND;
                 uint8_t fadeAmount = std::min(255.0f, agePercent * 256);
                 colorHighlight.fadeToBlackBy(fadeAmount);

```

Comment:
This was always under-drawing by 1, but once I fixed the Bresenham impl, I had to fix this as it should have been in the first place